src/Vocup/Forms/PracticeDialog.cs

```diff
@@ -26,7 +26,9 @@ public partial class PracticeDialog : Form
 
     private bool check;
     private bool solution;
-
+    private Random random;
+    private PracticeMode currentPracticeMode;
```

Comment:
Please introduce a new enum `PracticeDirection` to differentiate between a practice mode which can also be random and the actual practice direction.